src/schema/relations/type-helpers.ts

```
@@ -1,8 +1,21 @@
+import type {
+  InferSchemaOutput,
+  InferSchemaRelations,
+  SchemaInputWithId,
+} from "../type-helpers";
+import type {
+  AnyMonarchRelation,
+  MonarchNullableRelation,
+  MonarchOptionalRelation,
+  MonarchRelation,
+} from "./base";
+import { MonarchMany } from "./many";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)